cmd/tailscale/cli/ping.go

```diff
@@ -59,6 +60,24 @@ relay node.
 	})(),
 }
 
+func init() {
+	ffcomplete.Args(pingCmd, func(args []string) ([]string, ffcomplete.ShellCompDirective, error) {
```

Comment:
This is the same as the `nc` command - we should make this re-usable (and support IPs?)